src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,21 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.3.0
+
+### New Feature
+
+- Git Catalog feature will be part of the core of Okteto
+
+### Bug Fix
+
+- Use case insensitive check for GitHub ID
+- GitHub redeploy was having issues with the `repo` field in some cases
+- Support for "resources" for the okteto daemonset
+- Fix blank namespaces
+- Fix selecting a repository in the redeploy dialog
+- Show updated time instead of creation time in git deploys
```

Comment:
` git deploys` is an internal thing. we should say something like `Fix bug in the Okteto UI for the updated time of development environments`